src/styles/summernote-lite.scss

```diff
@@ -81,3 +81,14 @@
     display:inline-block;
   }
 }
+
```

Comment:
Good work so far. Thank you for adding the changes. Do you think you might be able to add the icon so it's done the same way the build system does it just so we don't have errant image files included, this way we can keep additions like this like the current method of including icons?

Reply:
https://github.com/summernote/summernote/pull/4072/commits/c1bd121e4dc95c9a5e9cdbd71bcfd1ddc5ee1857#diff-351334d016b7a57f4735d16ab9ac4a6f2277120aad5a422f62ec87537b35e625
I have added the icon to this PR. You can keep additions like this like the current method of including icons.